src/components/button.scss

```diff
@@ -8,120 +9,189 @@
 
   justify-content: center;
 
-  transition: all 120ms;
+  transition: all var(--rm-button-transition);
 
-  border: 0;
-  border-width: var(--rm-border-width);
   border-radius: var(--rm-radius-large);
+
   padding: var(--rm-space-small) var(--rm-space-medium);
```

Comment:
The default padding on the buttons seems high to me. The buttons seem larger than what I think looks good. The `small` is closer to what I would imagine a default looking like. 

look at github 

<img width="544" alt="image" src="https://user-images.githubusercontent.com/44909026/175330027-3f8bda91-8685-47be-b871-f61f9f170db9.png">

or LandOne mockups

<img width="541" alt="image" src="https://user-images.githubusercontent.com/44909026/175330124-81b6693b-6be5-46cd-b9af-0fdd8173e87c.png">


Both those have 32px whereas I think in this repo the height comes out 44px

Reply:
See other comment